qulice-checkstyle/src/test/resources/com/qulice/checkstyle/LocalVariableNames.java

```diff
@@ -35,6 +37,7 @@ void invalid() {
         try {
             int prolongations = 0;
```

Comment:
@jrdalpra can you also add test case proving that e.g. `very_long_variable_id` would be rejected?